core-domain/src/main/kotlin/com/svenjacobs/app/leon/core/domain/sanitizer/jdoqocy/JdoqocyRedirectSanitizer.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Léon - The URL Cleaner
+ * Copyright (C) 2022 Sven Jacobs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.svenjacobs.app.leon.core.domain.sanitizer.jdoqocy
+
+import android.content.Context
+import com.svenjacobs.app.leon.core.common.regex.RegexFactory
+import com.svenjacobs.app.leon.core.domain.R
+import com.svenjacobs.app.leon.core.domain.sanitizer.Sanitizer
+import com.svenjacobs.app.leon.core.domain.sanitizer.SanitizerId
+import com.svenjacobs.app.leon.core.domain.sanitizer.SearchResultSanitizer
+
+class JdoqocyRedirectSanitizer : SearchResultSanitizer(
```

Comment:
Thank you very much for this PR 👍🏼 

Regarding the name of this sanitizer, I propose to only name it `JdoqocySanitizer` and also adjust the names in the XMLs. `YoutubeRedirectSanitizer` was named this way because it handles `youtube.com/redirect` URLs. However most of the `SearchResultSanitizer` based sanitizers don't have `Redirect` in their name.

Reply:
Of course, it makes sense. 💪